layersvt/api_dump.h

```diff
@@ -45,7 +45,9 @@
 #include <algorithm>
 #include <chrono>
 #include <fstream>
+#include <string_view>
```

Comment:
nit: can we wrap these includes with `#ifdef __ANDROID__` 

I plan to got through this file and remove a lot of these that are not being used (after realizing it)

so

```c++
#ifdef ANDROID
#include <memory>
#include <string_view>

#include <android/log.h>
#include <sys/system_properties.h>
```

Reply:
Done.